src/pages/history/history.jsx

```diff
@@ -0,0 +1,23 @@
+import { VideoCard } from "components"
+import { useHistory, useVideo } from "contexts"
+import { useEffect } from "react"
+import "./history.css"
+
+const History = () => {
+    const {state} = useVideo()
+    const {getHistory,clearHistory} =useHistory()
+    useEffect(()=>{
+        getHistory()
+    })
+    return (
+    <div className="history-container center flex-c">
+        <button onClick={()=>clearHistory()} className="btn btn-with-icon">
```

Comment:
Suggestion: you can directly write `onClick=clearHistory}` (as it has no params), instead of adding a callback function.